### PR TITLE
Add logs terminal color and size

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
- This would add the terminal blue color.
- This would add the modal-xl size.

### CODE REVIEW

**Review by avishnu — APPROVED** [review]
Lgtm

File: scss/core/abstracts/_colors.scss [hunk 1]
```diff
@@ -93,6 +93,9 @@ $teal: #24A4B1 !default;
 
 $dodger-blue: #A3BAE0 !default;
 
+// Color used in terminal header and footers
+$terminal-blue: #3D51AD !default;
```
Comment (Pallavi-PH) [inline scss/core/abstracts/_colors.scss:97]: add color in `colors.js` and `colors.mdx` file as well. This is to display in the colors screens https://awesome-thompson-1f5f82.netlify.app/#/products-colors
Screenshot 2020-11-26 at 11 54 50 AM

File: docs/products/colors.mdx [hunk 2]
```diff
@@ -57,6 +57,9 @@ import ColorBlock from './components/colorblock.js'
   </div>
   <div className="col-sm-3">
     <ColorBlock name={dodgerblue} />
+  </div>
+    <div className="col-sm-3">
+    <ColorBlock name={terminalblue} />
```
Comment (Pallavi-PH) [inline docs/products/colors.mdx:62]: import this color at the beginning of the page

File: docs/utils/colors.js [hunk 3]
```diff
@@ -65,4 +65,7 @@ export const teal = [
 ];
 export const dodgerblue = [
   {'name':'dodger-blue', 'hax':'#A3BAE0', 'rgb':''},
+];
+export const terminalblue = [
+  {'name':'terminal-blue', 'hax':'#3D51AD', 'rgb':''},
```
Comment (Pallavi-PH) [inline docs/utils/colors.js:70]: Add this color at the beginning of this file in the comments

File: docs/products/colors.mdx (hunk 2 above)
Comment (Abhinandan-Purkait) [reply docs/products/colors.mdx:62]: - Added

File: docs/utils/colors.js (hunk 3 above)
Comment (Abhinandan-Purkait) [reply docs/utils/colors.js:70]: - Added

**Review by Pallavi-PH — APPROVED** [review]
LGTM

**Pull request merged**